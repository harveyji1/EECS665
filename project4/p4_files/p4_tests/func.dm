fn: (x: int, z: bool) int{
    a : int = 0;
}