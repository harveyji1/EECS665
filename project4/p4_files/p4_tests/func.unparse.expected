fn{(int,bool)->int} : (x{int} : int, z{bool} : bool) int  {
    a{int} : int = 0;
}
